### ISSUE TITLE
Full book name option has sometimes longname

### ISSUE THREAD

**Issue by tomtailor1983** [AntonyCorbett/OnlyV#27]
- Start OnlyV (ePub: Germany, Bible 2013)
- Choose a text in: 1. Mose or Psalm
- OnlyV will generates: 1.Mose (Genesis) 2:2 or Psalmen 119:1
- Correct would be: 1. Mose 2:2 or Psalm 119:1

**Comment by AntonyCorbett** [comment]
@tomtailor1983 The full names of books are extracted from the Bible contents page in the epub, so simply appear in OnlyV as they are listed.

**Comment by tomtailor1983** [comment]
Ok, but I think it could still be a bug. :) The psalm, for example, appears in the plural.
Here are two examples:

![grafik](https://user-images.githubusercontent.com/43112427/57580985-0d23e380-74b1-11e9-9e46-0578e7a392dc.png)

![grafik](https://user-images.githubusercontent.com/43112427/57580993-1f058680-74b1-11e9-9c7c-578014309e86.png)

Thank you, Antony!

**Comment by AntonyCorbett** [comment]
@tomtailor1983 Thanks, but as I said, the names of the books are simply extracted from the epub contents page. So the book titles for the above are "Psalmen" and "Mose (Genesis)". OnlyV uses the book name followed by chapter and verse. If this is inappropriate in the German language (as appears to be the case), then we need to find the correct title formulation somewhere in the epub.

**Comment by tomtailor1983** [comment]
Yes, that's what I mean. Maybe we can pull the title somewhere else from the epub. Thank you!

**Comment by richfrantz** [comment]
On the other hand, isn't the epub sourced from the Branch? I would think you should contact them for clarification. They'll either thank you for finding an error, or explain why it is correct.
________________________________

**Comment by AntonyCorbett** [comment]
@richfrantz I think @tomtailor1983 is right. The subtlety is only clearly exposed in non-English languages. However, we have one example in English - “Psalms” is the name of the book, but when we make reference to a verse it is more correctly “Psalm x:y”. No one has complained that OnlyV renders it “Psalms x:y” because we commonly use that formulation too. But I guess in German the use of the plural in this context is quite jarring.

**Comment by richfrantz** [comment]
I agree. My point is that if you are getting your data from the epub, the epub might have a bug.
If you go to wol.jw.org and switch to German, the Bible book is Psalmen, plural, but click inside and it reverts to the singular, Psalm.

Rich Frantz

________________________________

**Comment by malexarc** [comment]
Hi, is possible to add the ability to edit the book name as we are allow verse editing  please?
Thank you so much for your great work,
